frontend/src/components/dashboard/DashboardTitle.vue

```diff
@@ -21,8 +22,9 @@ const props = defineProps({
                 class="dashboard-button"
                 label="Grant new access"
                 @click="router.push('/grant')"
-                ><Icon icon="AddIcon" medium class="icon-size-1"
-            /></Button>
+                >
+                <Add />
```

Comment:
Same comment as previous,  maybe I am confused, is it better to use Icon component?